proto/launch/request.proto

```diff
@@ -14,3 +16,7 @@ message Request {
   google.protobuf.Any content = 5 [(cosmos_proto.accepts_interface) = "RequestContent"];
 }
 
+message ContentRemoveValidator {
```

Comment:
I would call it `ValidatorRemoval`

The content for `MsgRequestAddValidator` will be `GenesisValidator` and so it could be confusing to have the content prefix for some and not for others

Reply:
sure, I didn't like `ContentRemoveValidator` as well, but I didn't found a better name